ACE/ace/Service_Config.cpp

```diff
@@ -299,13 +299,13 @@ ACE_Service_Config::open_i (const ACE_TCHAR program_name[],
   // user forgets to define an object of type ACE_Service_Config).
   ACE_Service_Repository::instance (ACE_Service_Gestalt::MAX_SERVICES);
 
+#if !defined (ACE_LACKS_UNIX_SIGNALS)
```

Comment:
Retrieving reactor is not dependent on unix signals, concerned that this break things